seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/IcebergCatalogLoader.java

```diff
@@ -170,11 +169,10 @@ public static void doKerberosAuthentication(
                         "Start Kerberos authentication using principal {} and keytab {}",
                         principal,
                         keytabPath);
-                sun.security.krb5.Config.refresh();
```

Comment:
Is this logic invalid?

Reply:
> Is this logic invalid?

@hailin0   when using loginUserFromKeytabAndReturnUGI, refreshKrb5Config=true will be added by default. So I think there is no need to call refresh

Reply:
If your application is running and you modify the krb5.conf file at the same time. LoginUserFromKeytabAndReturnUGI not reload the changes. It will only read and load the current Kerberos configuration when the application starts.

Reply:
> If your application is running and you modify the krb5.conf file at the same time. LoginUserFromKeytabAndReturnUGI will not reload the changes. It will only read and load the current Kerberos configuration when the application starts.
> 
> I think the correct way to modify it is to find out why this error is reported instead of directly removing it

@dailai Please explain the impact of removing it

Reply:
> If your application is running and you modify the krb5.conf file at the same time. LoginUserFromKeytabAndReturnUGI will not reload the changes. It will only read and load the current Kerberos configuration when the application starts.
> 
> I think the correct way to modify it is to find out why this error is reported instead of directly removing it

@dailai https://github.com/apache/incubator-uniffle/issues/625 , It would be best to explain the necessity of the existence of this code

Reply:
> > If your application is running and you modify the krb5.conf file at the same time. LoginUserFromKeytabAndReturnUGI will not reload the changes. It will only read and load the current Kerberos configuration when the application starts.
> > I think the correct way to modify it is to find out why this error is reported instead of directly removing it
> 
> @dailai [apache/incubator-uniffle#625](https://github.com/apache/incubator-uniffle/issues/625) , It would be best to explain the necessity of the existence of this code

It's wight. You can refer this way https://github.com/apache/incubator-uniffle/pull/932

Reply:
> > > If your application is running and you modify the krb5.conf file at the same time. LoginUserFromKeytabAndReturnUGI will not reload the changes. It will only read and load the current Kerberos configuration when the application starts.
> > > I think the correct way to modify it is to find out why this error is reported instead of directly removing it
> > 
> > 
> > @dailai [apache/incubator-uniffle#625](https://github.com/apache/incubator-uniffle/issues/625) , It would be best to explain the necessity of the existence of this code
> 
> It's wight. You can refer this way [apache/incubator-uniffle#932](https://github.com/apache/incubator-uniffle/pull/932)

@dailai  Why not call Config.refresh() JDK>8?   So isn't this necessary?   

Have 2 ways to solve the issue.

- Remove sun.security.krb5.Config.refresh(); code.
- Add --add-exports java.security.jgss/sun.security.krb5=ALL-UNNAMED in the pom.xml of the project.

Firstly, explain the necessity of calling refresh clearly

Reply:
My doubt is the method called by loginUserFromKeytabAndReturnUGI which can automatically refresh krb5 conf. If you can prove that it is, I think you directly remove this line of code that is no problem.

Reply:
I mean that. @hailin0 It's right.
org.apache.seatunnel.connectors.seatunnel.iceberg.IcebergCatalogLoader#doKerberosAuthentication 
org.apache.hadoop.security.UserGroupInformation#loginUserFromKeytab
org.apache.hadoop.security.UserGroupInformation#loginUserFromKeytabAndReturnUGI
org.apache.hadoop.security.UserGroupInformation#doSubjectLogin
org.apache.hadoop.security.UserGroupInformation#newLoginContext
org.apache.hadoop.security.UserGroupInformation#newLoginContext
javax.security.auth.login.LoginContext#init
org.apache.hadoop.security.UserGroupInformation.HadoopConfiguration#getAppConfigurationEntry
org.apache.hadoop.security.UserGroupInformation.HadoopConfiguration#getKerberosEntry
which made refreshKrb5Config is true

Finally,  org.apache.hadoop.security.UserGroupInformation.HadoopLoginContext#login
call the method named com.sun.security.auth.module.Krb5LoginModule#login
![image](https://github.com/user-attachments/assets/ea526376-5287-481f-8c04-bfcf33dad938)